task_1/1.js

```diff
@@ -0,0 +1,15 @@
+/*
+Лизе нужно написать функцию sayHello(name), которая возвращает приветствие для пользователя. 
+Тем не менее, она влюблена в пользователя с именем "Mark", и хотела бы поприветствовать его немного иначе.
+Помогите ей)
+Пример вызовов:
+sayHello("Oleg"); // Hello, Oleg!
+sayHello("Viktor"); // Hello, Viktor!
+sayHello("Mark"); // Hi, Mark!
+*/
+
+const sayHello = (name) => name === 'Mark' ? `Hi, ${name}!` : `Hello, ${name}!`;
```

Comment:
1) What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue
2) Try to rework your code to have a single return statement. Difference in your returns just in greeting word.